test/tests/notebooks/install_location_install.ipynb

```diff
@@ -0,0 +1,103 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%install-location $cwd/swift-modules"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Installing packages:\n",
+      "\t.package(url: \"https://github.com/mxcl/Path.swift\", from: \"0.16.1\")\n",
+      "\t\tPath\n",
+      "\t.package(url: \"https://github.com/JustHTTP/Just\", from: \"0.7.1\")\n",
+      "\t\tJust\n",
+      "With SwiftPM flags: []\n",
+      "Working in: /tmp/tmpn3fi0x3q\n",
+      "Fetching https://github.com/mxcl/Path.swift\n",
+      "Fetching https://github.com/JustHTTP/Just\n",
+      "Completed resolution in 1.80s\n",
+      "Cloning https://github.com/mxcl/Path.swift\n",
+      "Resolving https://github.com/mxcl/Path.swift at 0.16.2\n",
+      "Cloning https://github.com/JustHTTP/Just\n",
+      "Resolving https://github.com/JustHTTP/Just at 0.7.1\n",
+      "Compile Swift Module 'Path' (9 sources)\n",
+      "Compile Swift Module 'Just' (1 sources)\n",
+      "Compile Swift Module 'jupyterInstalledPackages' (1 sources)\n",
+      "Linking ./.build/x86_64-unknown-linux/debug/libjupyterInstalledPackages.so\n",
+      "Initializing Swift...\n",
+      "Loading library...\n",
+      "Swift initialization complete!\n"
+     ]
+    }
+   ],
+   "source": [
+    "// Skip this cell or comment these lines after a successful install\n",
+    "%install '.package(url: \"https://github.com/mxcl/Path.swift\", from: \"0.16.1\")' Path\n",
```

Comment:
Use `%install '.package(path: "$cwd/SimplePackage")' SimplePackage` instead of installing from github, so that the test does not depend on github.